lib/Client.php

```diff
@@ -156,6 +161,12 @@ public function readLine($limit = null) {
             ));
         }
         $state = $this->state;
+        if (!$state->readOperations && false !== ($eolPos = \strpos($state->buffer, \PHP_EOL))) {
```

Comment:
fixed in a new commit, have a look you might have a better idea how to code it, it's not particularly readable at this point